Project_source/app/src/main/java/android/clicker/school_live_simulator/ComputerCourseState.kt

```diff
@@ -0,0 +1,30 @@
+package android.clicker.school_live_simulator
+
+abstract class ComputerCourseState: StudyCourseState {
+    protected abstract val price: Int
+
+    /**
+     * This is a list of available web tasks
+     * These types are described in enumeration class WebTask
+     */
+
+    protected val available_web_task: List<WebTask> = listOf<WebTask>()
+
+    /**
+     * This is a function for buying courses
+     * @param student Player who buys course
+     */
+
+    abstract fun buyNextCourse(student: Player)
```

Comment:
Same thing for every class
```suggestion
    abstract fun buyNextCourse(courses: Player.Courses)
```

Reply:
Fixed